app/templates/components/add-reminder.hbs

```diff
@@ -1,4 +1,3 @@
-{{!-- {{outlet}} --}}
 <form class="add-reminder-form" {{action 'addReminder' on="submit"}}>
   <label>
     Title:
```

Comment:
Can't click on the line for some reason but can we clean up the spacing on line 14 - where you define the class for the button